Makefile

```diff
@@ -259,13 +259,22 @@ amazon-cluster-manifest:
 	@pach-deploy amazon $(BUCKET_NAME) $(AWS_ID) $(AWS_KEY) $(AWS_TOKEN) $(AWS_REGION) $(STORAGE_NAME) $(STORAGE_SIZE)
 
 amazon-cluster:
-	aws s3api create-bucket --bucket $(BUCKET_NAME) --region $(AWS_REGION)
+	aws s3api create-bucket --bucket $(BUCKET_NAME) --create-bucket-configuration LocationConstraint=$(AWS_REGION)
```

Comment:
See the amazon reference here http://docs.aws.amazon.com/cli/latest/reference/s3api/create-bucket.html
The `--create-bucket-configuration LocationConstraint=$(AWS_REGION)` is recommended by aws if we use the region besides the default region `us-west-1`. 
![image](https://cloud.githubusercontent.com/assets/20234415/17610585/e77b9016-6071-11e6-870c-221b03ef8075.png)

Also the guide doesn't give an option named '--region <regionName>'., even sometimes `--region <regionName>` still can be used.
![image](https://cloud.githubusercontent.com/assets/20234415/17610675/d18bccfc-6072-11e6-80fc-572c5d7af7d8.png)

In conclusion, I recommend to adapt the script to `--create-bucket-configuration LocationConstraint=$(AWS_REGION)`